stdlib/Distributed/src/cluster.jl

```diff
@@ -224,6 +224,7 @@ It does not return.
 """
 start_worker(cookie::AbstractString=readline(stdin); kwargs...) = start_worker(stdout, cookie; kwargs...)
 function start_worker(out::IO, cookie::AbstractString=readline(stdin); close_stdin::Bool=true, stderr_to_stdout::Bool=true)
+    myrole!(:worker)
```

Comment:
Move this to `init_worker`. Custom transports can implement their own `start_worker` but still need to call `init_worker`.

Reply:
Done.